integrations/docker/images/chip-build-tizen/tizen-sdk-installer/install.sh

```diff
@@ -212,12 +216,14 @@ function install_tizen_sdk() {
         'parcel-0*.armv7l.rpm'
         'parcel-devel-*.armv7l.rpm'
         'pkgmgr-info-*.armv7l.rpm'
+        'sensord-devel-*.armv7l.rpm'
+        'sensord-dummy-*.armv7l.rpm'
         'vconf-compat-*.armv7l.rpm'
         'vconf-internal-keys-devel-*.armv7l.rpm')
     download "$URL" "${PKG_ARR[@]}"
 
     # Unified packages (snapshots)
-    URL="http://download.tizen.org/snapshots/tizen/unified/latest/repos/standard/packages/armv7l/"
+    URL="http://download.tizen.org/snapshots/TIZEN/Tizen-$TIZEN_VERSION/Tizen-$TIZEN_VERSION-Unified/latest/repos/standard/packages/armv7l/"
     PKG_ARR=(
         'capi-network-nsd-*.armv7l.rpm'
         'capi-network-thread-*.armv7l.rpm'
```

Comment:
Please add the following packages from [link](	
https://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/latest/repos/standard/packages/armv7l/) to tizen SDK to add changes related to modification of Bluetooth Tizen API. 
``` 
"capi-network-bluetooth-0"
"capi-network-bluetooth-devel-0"
``` 
After linking an app with the libraries above to run Tizen applications on the target device, there have to be upgraded packages: 
```
bluez-debuginfo-5* 
bluez-plugin-headless-5* 
bluez-profile_common-5* 
bluetooth-frwk-service-0* 
bluetooth-frwk-0* 
bluetooth-frwk-profile_common-0* 
bluetooth-frwk-core-0* 
bluetooth-frwk-oal-0* 
capi-network-bluetooth-0* 
```

Reply:
I've added capi-network-bluetooth-devel (with dependencies required to build and link application) to the SDK. However, I think that Bluetooth framework update can be done later, along with QEMU btvirt support. Otherwise, we will have new Bluetooth framework on QMEU but without a possibility to tests it...

Reply:
ok